docs/getting-started/getting-started.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: What you can use Patrol for?
+---
+
+# What you can use Patrol for?
+
+Patrol consists of 2 packages:
+
+- [patrol] – the core package
+- [patrol_cli] - the CLI app to run tests.
```

Comment:
Currently it's:

```
Patrol consists of 2 packages:

[patrol] – the core package
[patrol_cli] - the CLI app which enables the [native automation feature]

...

[native automation feature]: https://patrol.leancode.co/native/overview
```

I think that `the CLI app to run tests` is not specific enough. Maybe:

```
the CLI program to run tests that use the [native automation feature]
```

or (but sounds a bit strange)

```
the CLI program to run tests using [native automation feature]
```